css/index.css

```diff
@@ -4,10 +4,13 @@
 }
 
 body {
-  background-color: rgb(187, 247, 187);
+  background-image: url(../img/cal.jpg) ;
```

Comment:
I think a calculator doesn't need that kind of BG image. Giving it a aesthetic simple color background will be good enough.